components/org.wso2.carbon.business.rules.core/src/main/java/org/wso2/carbon/business/rules/core/api/impl/BusinessRulesApiServiceImpl.java

```diff
@@ -47,28 +47,27 @@
     private static final Logger log = LoggerFactory.getLogger(BusinessRulesApiServiceImpl.class);
 
     @Override
-    public Response createBusinessRule(String businessRule, Boolean deploy
+    public Response createBusinessRule(String businessRule, Boolean shouldDeploy
     ) throws NotFoundException {
         TemplateManagerService templateManagerService = TemplateManagerInstance.getInstance();
 
         // convert the string received from API, as a json object
         Gson gson = new GsonBuilder().disableHtmlEscaping().setPrettyPrinting().create();
         JsonObject businessRuleJson = gson.fromJson(businessRule, JsonObject.class);
         int status;
-
         // Check the business rule type of the json object
         if (businessRuleJson.get("type").toString().equals("\"" + TemplateManagerConstants
```

Comment:
Use constant for "type", and try to reuse.

Reply:
Fixed by f5b95d2e8cbf3ac583081822033d065ab10780f5